test/service/test_response.py

```diff
@@ -1353,8 +1347,8 @@ def test_ranged_values(self):
                     ],
                     "donorCount": 4,
                     "organismAge": [
-                        "45-49",
-                        "65-69"
+                        {"value": "45-49", "unit": "year"},
```

Comment:
… by this?

Reply:
I don't think so. The [other test ](https://github.com/DataBiosphere/azul/pull/2031#discussion_r537957134)makes requests with different sets of filters on the `organismAgeUnit`, then it asserts that the hits in the response contain the filtered field. Since we now have `unit` in `organismAge` we removed `organismAgeUnit` from the hits response, but it is still a valid facet, no? 
But independently of the format changes, filtering on a not-None and a None fields would still require us to iterate through the response and validate that the filter worked. But I might be missing something.

Reply:
Let's discuss in parking lot.

Reply:
@hannes-ucsc: "Ignore this complaint, it's not justified."